root/variables.tf

```diff
@@ -19,3 +19,9 @@ variable "custom_tags" {
   type    = map
   default = {}
 }
+
+variable "records" {
+  type        = map
+  default     = {}
+  description = "Map or records to add to the dns zone"
```

Comment:
Map *of* records to add to the dns zone. Could you also add  this to the other file in subdomain/variables.tf please ?